tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/wizard/NewLiferayPluginProjectWizard.java

```diff
@@ -154,23 +160,39 @@ protected void performPostFinish()
     {
         super.performPostFinish();
 
+        final List<IProject> projects = new ArrayList<IProject>();
+
         final NewLiferayPluginProjectOp op = element().nearest( NewLiferayPluginProjectOp.class );
-        final IProject project = CoreUtil.getProject( op.getFinalProjectName().content() );
 
-        try
+        ElementList<ProjectName> projectNames = op.getProjectNames();
+
+        for( ProjectName projectName : projectNames )
         {
-            addToWorkingSets( project );
+            final IProject newProject = CoreUtil.getProject( projectName.getName().content() );
+
+            if( newProject != null )
+            {
+                projects.add( newProject );
+            }
         }
-        catch( Exception ex )
+
+        for( final IProject project : projects )
         {
-            ProjectUI.logError( "Unable to add project to working set", ex );
+            try
+            {
+                addToWorkingSets( project );
+            }
+            catch( Exception ex )
+            {
+                ProjectUI.logError( "Unable to add project to working set", ex );
+            }
         }
 
-        openLiferayPerspective( project );
+        openLiferayPerspective( projects.get(0) );
```

Comment:
Lets add the projects.get(0) to a local final variable so we don't call it 5 times.